develop/blockchains/get-started/intro-polkadot-sdk.md

```diff
@@ -0,0 +1,86 @@
+
+Every blockchain platform relies on a decentralized network of computers—called nodes—that communicate with each other about transactions and blocks.
+In general, a node in this context is the software running on the connected devices rather than the physical or virtual machine in the network.
+As software, Substrate-based nodes consist of two main parts with separate responsibilities:
+
+- **Client** - services to handle network and blockchain infrastructure activity
+    - Native Binary
+    - Executes the Wasm runtime
+    - Everything else: Database, Networking, Mempool, Consensus, and more
+    - Also known as Host
+- **Runtime** - business logic for state transitions
+    - Application logic
+    - Wasm
+    - Stored as a part of the chain state
+    - Also known as STF (State Transition Function)
+
+### FRAME
+
+FRAME provides the core modular and extensible components that make the Substrate SDK flexible and adaptable to different use cases.
+FRAME includes Rust-based libraries that simplify the development of application-specific logic.
+Most of the functionality that FRAME provides takes the form of plug-in modules called [pallets](/polkadot-protocol/glossary#pallet) that you can add and configure to suit your requirements.
+
+### Cumulus
+
+Cumulus provides utilities and libraries to turn FRAME-based runtimes into runtimes that can be a parachain on Polkadot. Cumulus runtimes are still FRAME runtimes but contain the necessary functionality that allows for that runtime to become a parachain on a relay chain.
+
+## Why Use Polkadot SDK?
+
+Using the Polkadot SDK, you can build application-specific blockchains without the complexity of building a blockchain from scratch or the limitations of building on a general-purpose blockchain.
+You can focus on crafting the business logic that makes your chain unique and innovative with the additional benefits of flexibility, upgradeability, open-source licensing, and cross-consensus interoperability.
+
+## How to Create a Custom Blockchain using the SDK?
+
+Before starting your blockchain development journey, you'll need to decide whether you want to build a standalone chain or a parachain that connects to the Polkadot network. Each path has its considerations and requirements. Once you've made this decision, follow these development stages:
+
+```mermaid
+graph LR
+    A[Install] --> B[Build]
+    B --> C[Deploy]
+```
+
+1. [Install the Polkadot SDK](./install-polkadot-sdk.md) - set up your development environment with all necessary dependencies and tools
+
+2. [Build the Chain](TODO: update-path) - learn how to create and customize your blockchain's runtime, configure pallets, and implement your chain's unique features
+
+3. [Deploy the Chain](./deploy-blockchain-to-polkadot.md) - follow the steps to launch your blockchain, whether as a standalone network or as a parachain on Polkadot
+
```

Comment:
```suggestion
1. [Install the Polkadot SDK](./install-polkadot-sdk.md){taget=\_blank} - set up your development environment with all necessary dependencies and tools

2. [Build the Chain](TODO: update-path){target=\_blank} - learn how to create and customize your blockchain's runtime, configure pallets, and implement your chain's unique features

3. [Deploy the Chain](./deploy-blockchain-to-polkadot.md){target=\_blank} - follow the steps to launch your blockchain, whether as a standalone network or as a parachain on Polkadot

```

Reply:
I don't think we actually want to use `target=\_blank` here because we want users to go to the other page